mysql-test/main/mdev-32395.test

```diff
@@ -0,0 +1,13 @@
+create table t1 (c1 int) engine=myisam;
```

Comment:
I see this is draft, but based on MDEV comments it should be almost ready to merge.

Test in existing file `mysql-test/main/derived_cond_pushdown.test` and include  header indicating the MDEV /title that is being fixed (before "End of 10.4 tests").

don't need to be explict about `engine=myisam`.